Library/ViewModels/CommentRepliesViewModel.swift

```diff
@@ -162,9 +162,21 @@ public final class CommentRepliesViewModel: CommentRepliesViewModelType,
         CommentsViewModel.postCommentProducer
       )
 
-    let repliesAndTotalCount = replies.withLatestFrom(totalCountProperty.signal)
+    let errorLoadingMoreReplies =
+      error
+        .withLatestFrom(totalCountProperty.signal)
+        .combineLatest(with: project)
+        .map { errorAndTotalCount, project -> (([Comment], Int), Project) in
+          let (_, totalCount) = errorAndTotalCount
+          return (([], totalCount), project)
+        }
 
-    self.loadRepliesAndProjectIntoDataSource = Signal.combineLatest(repliesAndTotalCount, project)
+    let repliesAndTotalCount = replies.withLatestFrom(totalCountProperty.signal).combineLatest(with: project)
+
+    self.loadRepliesAndProjectIntoDataSource = Signal.merge(
+      repliesAndTotalCount,
+      errorLoadingMoreReplies
```

Comment:
`repliesTotalCountAndProject`
`erroredRepliesTotalCountAndProject`

Reply:
Ended up deleting both of these constants in the re-factor.